x/nameservice/keeper/grpc_query_test.go

```diff
@@ -0,0 +1,269 @@
+package keeper_test
+
+import (
+	"context"
+	"fmt"
+	"github.com/tharsis/ethermint/x/nameservice/client/cli"
+	nameservicetypes "github.com/tharsis/ethermint/x/nameservice/types"
+	"os"
+)
+
+func (suite *KeeperTestSuite) TestGrpcQueryParams() {
+	grpcClient := suite.queryClient
+
+	testCases := []struct {
+		msg string
+		req *nameservicetypes.QueryParamsRequest
+	}{
+		{
+			"Get Params",
+			&nameservicetypes.QueryParamsRequest{},
+		},
+	}
+	for _, test := range testCases {
+		suite.Run(fmt.Sprintf("Case %s ", test.msg), func() {
+			resp, _ := grpcClient.Params(context.Background(), test.req)
+			defaultParams := nameservicetypes.DefaultParams()
+			suite.Require().Equal(defaultParams.String(), resp.GetParams().String())
+		})
+	}
+}
+
+func (suite *KeeperTestSuite) TestGrpcGetRecordLists() {
+	grpcClient, ctx := suite.queryClient, suite.ctx
+	sr := suite.Require()
```

Comment:
Hmm this is creating some readability issues. Let's use it directly?

Reply:
Okay